packages/ui/src/components/Modal/index.tsx

```diff
@@ -343,12 +343,16 @@ export const Modal = memo(
               <StyledContainer>
                 {isClosable && (
                   <Button
+                    data-testid={
+                      dataTestId ? `${dataTestId}-close-button` : undefined
+                    }
                     onClick={onClose || onCloseCallBack}
-                    title="close"
-                    variant="transparent"
+                    variant="ghost"
+                    size="small"
                     icon="close"
-                    iconSize={20}
-                    action
+                    sentiment="neutral"
+                    aria-label="close"
+                    tooltip="close"
```

Comment:
As we have `aria-label` we do not need tooltip here. Previous title was used for a11y, screen reader will be capable of reading the `aria-label`